src/main/java/com/example/demo/FirstSpringBootApplication.java

```diff
@@ -11,6 +11,7 @@
     public static void main(String[] args) {
         System.out.println("new comment");
         SpringApplication.run(FirstSpringBootApplication.class, args);
+        System.out.println("new comment1");
```

Comment:
Can we please have a third comment? This is not enough comments :)

Reply:
Ok. here you go @MudassarBashir

Reply:
🥇